app/google-services.json

```diff
@@ -0,0 +1,46 @@
+{
+  "project_info": {
+    "project_number": "934985628718",
+    "firebase_url": "https://base-project-b3d16.firebaseio.com",
```

Comment:
Please invite us to this Firebase project  😉

Reply:
Done